feat(unlock-protocol-com): Add blog post for redesigned checkout (#9184)
unlock-protocol/unlock: unlock-protocol-com/blog/redesigning-our-checkout.md

---
title: The brand new checkout experience
authorName: searchableguy

> publishDate: July 5, 2022

Reviewer: ```suggestion
publishDate: July 6, 2022
```

description: We are launching public preview of new checkout experience.
image: /images/blog/redesigned-checkout/new-checkout.png
---

## Checkout experience

We've completely redesigned the Unlock checkout experience after a ton of user testing and feedback.

<video controls autoplay>
  <source src="/images/blog/redesigned-checkout/checkout-experience.mp4" type="video/mp4">
</video>

### Lock Selection

Memberships will be grouped by networks and provide information about the network to users. This move is intended to educate the users about the network and make it easier for creators to sell memberships on multiple networks with different benefits.

![lock selection on checkout](/images/blog/redesigned-checkout/new-checkout.png)

### Choosing Quantity

You will be able to select quantity and payment method you want to use earlier in the flow.

![quantity selection on checkout](/images/blog/redesigned-checkout/new-checkout-quantity.jpg)

### Recipient and metadata collection

> Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy memberships than allowed by the lock owner early in the flow.

Author: ```suggestion
Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy more memberships than allowed by the lock owner early in the flow.
```

> Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy memberships than allowed by the locker owner early in the flow.

Reviewer: ```suggestion
Recipients field will be populated based on the quantity. You will be able to add recipient addresses and fill out information. We've added more validation checks to ensure user cannot buy memberships than allowed by the lock owner early in the flow.
```

![metadata and recipients on checkout](/images/blog/redesigned-checkout/new-checkout-metadata.png)

### Payment using card

We've made it easier to pick and change card using the checkout. We're planning to introduce more traditional payment options soon.

![card selection on checkout](/images/blog/redesigned-checkout/new-checkout-add-card.png)

### Confirmation

> Our new confirmation page will show you how many memberships you are buying, what the total amount is, how many times a membership will recurr if it's a recurring membership and more.

Reviewer: ```suggestion
Our new confirmation page will show you how many memberships you are buying, what the total amount is, how many times a membership will be renewed if it's a recurring membership and more.
```
